src/com/palmergames/bukkit/towny/command/BaseCommand.java

```diff
@@ -174,4 +174,15 @@
 			return Collections.emptyList();
 		}
 	}
+
+	/**
+	 * Parses the given string into a boolean choice.
+	 * @param str The string to parse
+	 * @return true for "ON", false for "OFF", or null if no match.
+	 */
+	public static Boolean parseToggleChoice(String str) {

```

Comment:
I don't think this needs to be `public static`. All command classes should extend `BaseCommand` so this can just be `protected`.

Reply:
Also this is totally optional, but you can return `Boolean.TRUE` and `Boolean.FALSE` to remove any autoboxing. 
Am not the biggest fan of returning null for primitives, since it's very easy to hit an NPE that way. Alternatively this could accept a default value and resort to the default if it doesn't match.

Reply:
The thing is kinda, we use this method as a tri state boolean, if the boolean is set (true/false), use that as the choice for the toggle and if the boolean is null, we will default back to the old behaviour.

Reply:
I have replaced the Boolean value with an Optional of type Boolean, so that it's clear to the user who calls this method the result might be null.